### DISCUSSION TITLE
Is it possible to remove the New button from index page?

### DISCUSSION THREAD

**Discussion by rubabnaziri** [filamentphp/filament#2081] [General]
Is it possible to remove the New button from index page?
I have a permissions page. I just want to show them and not create them since they are created from backend.
I want to remove the New Edit and Delete buttons from the page.

**Comment by ryangjchandler** [comment]
@rubabnaziri You can overwrite the `canCreate()` method on the resource and `return false`. This should hide the button :)

**Reply by rubabnaziri** [reply]
Hey Ryan! Thanks for the quick response. I have tried it. CanCreate works on pages where we want to hide create button from relations table. This is the main index page. CanCreate isnt working there.

**Reply by danharrin** [reply]
You can just delete the page class and remove it from `getPages()`
